docs/best-practices/continuous-integration/github.md

```diff
@@ -0,0 +1,50 @@
+# GitHub Actions Integration
+
+Use [GitHub Actions runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) running on a macOS machine
+to build using fastlane.
```

Comment:
I think we don't break lines in our docs, so I'd join them:

```suggestion
Use [GitHub Actions runner](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners) running on a macOS machine to build using _fastlane_.
```

(I also italicized `fastlane` 😁 )